ord_schema/proto/reaction.proto

```diff
@@ -1006,6 +1006,7 @@ message Volume {
     MILLILITER = 1;
     MICROLITER = 2;
     LITER = 3;
+    NANOLITER = 4;
```

Comment:
Feel free to move this up in the list so it shows up in a logical order, just keep the current tag number.